web/gui-v2/src/components/DetailViewMoreMetadataDialog.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { Dialog, DialogTitle } from '@mui/material';
+
+import { ButtonStyled, ExternalLink } from '@eto/eto-ui-components';
+
+import TwoColumnTable from './TwoColumnTable';
+
+const styles = {
+  dialog: css`
+    .MuiPaper-root {
+      background-color: var(--bright-blue-light);
+    }
+  `,
+  dialogTitle: css`
+    font-family: GTZirkonRegular;
+    text-align: center;
+  `,
+  dialogInnerWrapper: css`
+    font-family: GTZirkonLight;
+    padding: 0 0.5rem 0.5rem;
+  `,
+  dialogContents: css`
+    background-color: var(--bright-blue-lightest);
+    max-height: calc(100vh - 200px);
+    overflow-y: auto;
+    padding: 0.5rem;
+  `,
+  table: css`
+    th,
+    td {
+      background-color: var(--bright-blue-lightest);
+    }
+  `,
+  linkWrapper: css`
+    display: flex;
+    flex-direction: column;
+  `,
+  dialogBottom: css`
+    display: flex;
+    justify-content: center;
+    margin-top: 0.5rem;
+  `,
+};
+
+/**
+ * A dialog for displaying more metadata about a particular company.
+ *
+ * @param {object} props
+ * @param {object} props.data
+ * @param {boolean} props.isOpen
+ * @param {(newState: boolean) => void} props.updateIsOpen
+ * @returns {JSX.Element}
+ */
+const MoreMetadataDialog = ({
+  data,
+  isOpen,
+  updateIsOpen,
+}) => {
+  const metadata = [
+    { title: 'Name', value: data.name },
+    { title: 'Aliases', value: data.aliases },
+    { title: 'Country', value: data.country },
+    { title: 'Continent', value: data.continent },
+    { title: 'Website', value: data.website ? <ExternalLink href={data.website}>{data.website}</ExternalLink> : undefined },
+    {
+      title: 'Crunchbase',
```

Comment:
@za158 Brian has put both the current company's crunchbase URL and its childrens' in the same field. I think that's fine/good, but it's a little different from what you described, so I wanted to make sure you were ok with it too

Reply:
I think it's OK - will be a lot in some cases but as long as it's hidden by default I'm not concerned